srcpkgs/gst-omx/patches/0001-ARM-build.patch

```diff
@@ -0,0 +1,120 @@
+From edbc89c33db360cd9cf27ea2231aba8ea0d8095d Mon Sep 17 00:00:00 2001
+From: Nathan Owens <ndowens@artixlinux.org>
+Date: Sun, 15 Nov 2020 13:34:42 -0600
+Subject: [PATCH] ARM build
```

Comment:
Can you explain more clearly what this patch is doing?

Reply:
seems to be a bunch of ugly hacks for the rpi target, i'd just drop it and use `-Dtarget=generic` universally (i.e. drop the whole `XBPS_TARGET_MACHINE` switch case)